datadog/dogstatsd/base.py

```diff
@@ -45,9 +49,12 @@
 DEFAULT_PORT = 8125
 
 # Buffering-related values (in seconds)
-DEFAULT_FLUSH_INTERVAL = 0.3
-MIN_FLUSH_INTERVAL = 0.0001
+DEFAULT_BUFFERING_FLUSH_INTERVAL = 0.3
+MIN_BUFFERING_FLUSH_INTERVAL = 0.0001
 
+# Aggregation-related values (in seconds)
+DEFAULT_AGGREGATION_FLUSH_INTERVAL = 2
+MIN_AGGREGATION_FLUSH_INTERVAL = 0.01
```

Comment:
Maybe this can be the same as `MIN_BUFFERING_FLUSH_INTERVAL`? (we could just keep `MIN_FLUSH_INTERVAL` as it was and use it for both?) 

Unless this new value has some significance. I don't see why the minimum couldn't be much smaller even if impractical.